front/src/common/Grid.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import cx from 'classnames';
+import './Grid.scss';
+
+type GridProps = {
+  data: string[];
+  filter?: string;
+  extraClass?: string;
+  updateData: (data: string) => void;
+  closeModal?: () => void;
+};
+
+function isFiltered(item: string, input: string): boolean {
+  return input !== '' && !item.toLocaleLowerCase().includes(input.toLocaleLowerCase());
+}
+
+function Grid({ data, filter, extraClass, updateData, closeModal }: GridProps) {
```

Comment:
Can you use a const here ? (it's a detail) 
```suggestion
const Grid = ({ data, filter, extraClass, updateData, closeModal }: GridProps) => {
```

Reply:
done